src/AnnotationDialog.js

```diff
@@ -569,106 +542,38 @@ class AnnotationDialog extends EventEmitter {
-        profileContainerEl.appendChild(createdEl);
-
-        // Comment
-        const commentTextEl = document.createElement('div');
-        commentTextEl.appendChild(textEl);
-        annotationEl.appendChild(commentTextEl);
-
-        // Delete button
-        if (!annotation.permissions.can_delete) {
-            return;
-        }
-
-        const deleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON_PLAIN, CLASS_BUTTON_DELETE_COMMENT],
-            this.localized.deleteButton,
-            ICON_DELETE,
-            constants.DATA_TYPE_DELETE
-        );
-        annotationEl.appendChild(deleteBtn);
-
-        const deleteConfirmEl = document.createElement('div');
-        deleteConfirmEl.classList.add(CLASS_DELETE_CONFIRMATION);
-        deleteConfirmEl.classList.add(constants.CLASS_HIDDEN);
-        annotationEl.appendChild(deleteConfirmEl);
-
-        const confirmMsgEl = document.createElement('div');
-        confirmMsgEl.classList.add(constants.CLASS_DELETE_CONFIRM_MESSAGE);
-        confirmMsgEl.textContent = this.localized.deleteConfirmation;
-        deleteConfirmEl.appendChild(confirmMsgEl);
-
-        const deleteBtnsEl = document.createElement('div');
-        deleteBtnsEl.classList.add(constants.CLASS_BUTTON_CONTAINER);
-        deleteConfirmEl.appendChild(deleteBtnsEl);
-
-        const cancelDeleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON, CLASS_CANCEL_DELETE],
-            this.localized.cancelButton,
-            this.localized.cancelButton,
-            constants.DATA_TYPE_CANCEL_DELETE
-        );
-        deleteBtnsEl.appendChild(cancelDeleteBtn);
-
-        const confirmDeleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON, CLASS_BUTTON_DELETE_CONFIRM, constants.CLASS_BUTTON_PRIMARY],
-            this.localized.deleteButton,
-            this.localized.deleteButton,
-            constants.DATA_TYPE_CONFIRM_DELETE
+        const locale = this.locale.substr(0, this.locale.indexOf('-'));
+
+        this.annotationListComponent = render(
+            <ul className='ba-annotation-list'>
+                {this.annotations.map((item) => {
+                    const { annotationID, created, modified, text, user, type } = item;
+                    if (type === constants.TYPES.highlight) {
+                        return null;
+                    }
+
+                    return (
+                        <li className='ba-annotation-list-item' key={`annotation_${item.annotationID}`}>
+                            <Annotation
+                                id={annotationID}
+                                createdBy={user}
+                                createdAt={created}
+                                modifiedAt={modified}
+                                message={text}
+                                locale={locale}
+                                onDelete={() => this.emitAnnotationDelete(item)}
+                                {...item}
```

Comment:
This means that the component receives both `created` and `createdAt`, for example. It seems like this could be cleaned up a bit.